### PR TITLE
Adds mobile_token to JSON sign in response if successful.

### PR DESCRIPTION
**Pull request by mafernando**
- Added lines to sessions controller that generate and persist a new token when user signs in via JSON.
- Added mobile_token attribute accessor to Staff model.
- Added mobile_token attribute to StaffSerializer.

Address issue #361

### CODE REVIEW

**Comment by mafernando** [comment]
Small note - if you are trying to sign in to API via JSON, you'll need to add a `.json` to the end of the sign in URL for it to be identified as JSON in API.
```
http://localhost:3000/staff/sign_in.json
```
Additionally, the staff email and password need to exist as JSON attributes within a JSON staff object.
```
{"staff":{"email":"admin@projectjellyfish.org","password":"jellyfish"}}
```

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@ def create
 
         if resource
           sign_in(resource_name, resource)
+          resource.mobile_token = resource.secret = SecureRandom.hex
```
Comment (stackus) [inline app/controllers/sessions_controller.rb]: Didn't like `api_token`?

Comment (mafernando) [inline app/controllers/sessions_controller.rb]: https://github.com/projectjellyfish/api/issues/361

Chris wanted mobile_token, so I decided to go with that.

Changing it seems simple enough going forward, so I'm not too concerned.

**Comment by mafernando** [comment]
@stackus - Chris went ahead with api_token :+1:

**Pull request merged**